src/go/rpk/pkg/cli/acl/acl.go

```diff
@@ -72,17 +72,34 @@ group.
 
 PRINCIPALS
 
-All ACLs require a principal. A principal is composed of two parts: the type
-and the name. Within Redpanda, only one type is supported, "User". The reason
-for the prefix is that a potential future authorizer may add support for
-authorizing by Group or anything else.
+All ACLs require a principal or a role. A principal comprises a user and type.
```

Comment:
```suggestion
All ACLs require a principal or a role. A principal is comprised of a user and type.
```

Reply:
This suggestion seems clearer than the original.

Reply:
Missed this. Fair point, just slightly non-standard usage for whatever that's worth. Would "composed of" have roughly the same effect?